src/store/timetravel.js

```diff
@@ -27,6 +27,9 @@ const filteredActions = [
   'project/setPreviousStoryVisible',
   'project/setGridVisible',
 ];
+const permanentActions = [
```

Comment:
We should consider adding map placement to this instead of waiting for map placement before initializing timetravel

Reply:
ooh, good idea. I'll do that tomorrow in this same PR.